### PR TITLE
Ares Protocol milestone 1

### PR DESCRIPTION
**Pull request by jiyilanzhou**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/w3f/Open-Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A BTC address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://gist.github.com/Noc2/75bc58e8ce9b5d419ff883b0cf2b8c19).
- [x] The total funding amount of the project is below CHF 30k at the time of submission.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

**Review by Noc2 — APPROVED** [review]
Thanks. Looks good to me. I will share it with the rest of the team.

File: applications/ares_protocol.md [hunk 1]
```diff
@@ -77,10 +77,10 @@ In this milestone, We will implement ares oracle proof-of-concept, A oracle pall
 | Number | Deliverable | Specification |
 | ------------- | ------------- | ------------- |
 | 1. | oracle pallet | requested data, generated events, and callbacks to data |
-| 2. | scanner | scanner parachain oracle request via block metadata, parse the specific request data |
+| 2. | scanner | scanner parachain oracle request via block event, parse the specific request data |
 | 3. | provider | data warehouse returns the correct request data use http request | 
 | 4. | Testing | This milestone will have unit-test for pallet impemented, simulated all functions. |
-| 5. | example for demonstration | Provide  parachain oracle pallet integrate example, It use golang implement scanner and provider, may use go-substrate-api|
+| 5. | example for demonstration | Provide  parachain oracle pallet integrate example, It replace golang with js, integrate videos and the front-end into the deliveries|
```
Comment (Noc2) [inline applications/ares_protocol.md:83]: Could you change "It replace golang with js" to "it uses the scanner, which is written in js"? And generally specify the programming languages of each delivery.

**Review by Noc2 — CHANGES_REQUESTED** [review]

File: applications/ares_protocol.md (hunk 1 above)
Comment (jiyilanzhou) [reply applications/ares_protocol.md]: I aleady has update it, and add some new worker.

Comment (Noc2) [reply applications/ares_protocol.md]: Thanks

**Review by Noc2 — APPROVED** [review]
Looks good. I will share it with the rest of the team.

**Review by laboon — APPROVED** [review]

**Review by mmagician — APPROVED** [review]

**Pull request merged**